pkg/router/controller/endpointsubset/converter.go

```diff
@@ -0,0 +1,63 @@
+package endpointsubset
+
+import (
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/api/discovery/v1beta1"
+)
+
+// ConvertEndpointSlice converts items to a slice of EndpointSubset's.
+func ConvertEndpointSlice(items []v1beta1.EndpointSlice, addressOrderByFuncs []EndpointAddressLessFunc, portOrderByFuncs []EndpointPortLessFunc) []v1.EndpointSubset {
```

Comment:
Perhaps if you always use the same "order by" function each time, you can remove it from the arguments and hard code it in the function.  Would make cleaner tests.

Reply:
Yep, good call. done in https://github.com/openshift/router/commit/26cbd10beee116fcf9f8054265a0e72eb213b5e1

Reply:
> Yep, good call. done in [26cbd10](https://github.com/openshift/router/commit/26cbd10beee116fcf9f8054265a0e72eb213b5e1)

Did these changes make it into `26cbd10`? I think I might still be seeing the original code?

Reply:
https://github.com/openshift/router/pull/231/commits/26cbd10beee116fcf9f8054265a0e72eb213b5e1#diff-eb4119cc4334b78b1a123bdf8b86d9898af5909d18b0becc0daac1c000f8ac81R100

Reply:
Ah I see, a middle ground was reached here. `ConvertEndpointSlice` still has the `addressOrderByFuncs` and `portOrderByFuncs` parameters, which is good in the event that we want to use different ordering funcs in the future. But the test cases however, don't specify specific `<foo>OrderByFuncs` and instead get tested with the defaults. Makes sense to me.

Reply:
That's fine by me.